rasa_core/policies/embedding_policy.py

```diff
@@ -1427,9 +1452,15 @@ def load(cls, path: Text) -> 'EmbeddingPolicy':
         if not os.path.exists(checkpoint + '.meta'):
             return cls(featurizer=featurizer)
 
+        config_file = os.path.join(path, "{}.config.pkl".format(file_name))
+
+        with io.open(config_file, 'rb') as f:
+            config = pickle.load(f)
```

Comment:
since we persist the whole config, let's pass it to the policy as a kwargs. Could you please add tests that loaded policy has new config

Reply:
I have not write any tests code, let me try.

Reply:
sorry, about adding tests I have no idea.